Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml

```diff
@@ -273,16 +457,238 @@ script:
       description: Alert created date
       type: date
     - contextPath: MsGraph.Alert.EventDate
-      description: Alert event date
+      description: Relevant only for V1. Alert event time
       type: date
     - contextPath: MsGraph.Alert.Status
       description: Alert status
       type: string
     - contextPath: MsGraph.Alert.Vendor
-      description: Alert vendor
+      description: Relevant only for V1. Alert vendor/provider
       type: string
     - contextPath: MsGraph.Alert.Provider
-      description: Alert provider
+      description: Relevant only for V1. Alert provider
```

Comment:
General comment:
All descriptions must end with a period (.)
Please add a period at the end of all descriptions.